common/metrics/metrics.go

```diff
@@ -1,14 +1,50 @@
 package metrics
 
 import (
+	"context"
+	"fmt"
 	"math"
+	"strings"
+	"sync"
 
+	dashboard "cloud.google.com/go/monitoring/dashboard/apiv1"
+	"cloud.google.com/go/monitoring/dashboard/apiv1/dashboardpb"
 	"github.com/GoogleCloudPlatform/spanner-migration-tool/common/constants"
 	"github.com/GoogleCloudPlatform/spanner-migration-tool/internal"
 	"github.com/GoogleCloudPlatform/spanner-migration-tool/internal/reports"
 	"github.com/GoogleCloudPlatform/spanner-migration-tool/proto/migration"
 )
 
+// Defines dimensions for Monitoring Dashboard Metrics
+const (
+	// Default height of a tile in the monitoring dashboard
+	defaultMonitoringMetricHeight int32 = 16
```

Comment:
I think this should be a relative height as opposed to absolute

Reply:
Hi Aditya, I am not sure I understand. Relative to what?